src/ng/directive/ngModel.js

```diff
@@ -518,7 +519,7 @@ var NgModelController = ['$scope', '$exceptionHandler', '$attrs', '$element', '$
 
     // Check if the there's a parse error, so we don't unset it accidentially
     var parserName = ctrl.$$parserName || 'parse';
```

Comment:
This variable doesn't seem to be used now.  What does this mean for the default of 'parse'?